### PR TITLE
Simplify client dependency resolution a bit.

### PR DESCRIPTION
**Pull request by justinfagnani**
Hopefully this avoids some Windows issues, like #694

 - [ ] CHANGELOG.md has been updated

### CODE REVIEW

**Comment by rictic** [comment]
Looks like we added appveyor but it's not configured properly?

Also we're getting 6.6.0-pre.3 for the client-side, which isn't in the acceptable version range for our client in package.json

File: runner/config.ts [hunk 1]
```diff
@@ -169,49 +138,51 @@ export function truncatePathToDir(directory: string, pathName: string): string|
...
   // package or its dependencies' dependencies
-  const absoluteWCTRoot =
-      truncatePathToDir(wctPackageName, absoluteBrowserPath);
+  const wctBrowserPackageDirPath = path.dirname(wctBrowserPackageMainPath);
   const resolvedEntrypoints: string[] = [];
 
   for (const npmPackage of npmPackages) {
     const absoluteNpmMainPath =
-        resolve.sync(npmPackage.name, {basedir: absoluteWCTRoot});
+        resolve.sync(npmPackage, {basedir: wctBrowserPackageDirPath});
```
Comment (e111077) [inline runner/config.ts]: Does this work when the filename is not the `main` in the package?

**Review by e111077 — CHANGES_REQUESTED** [review]

File: runner/config.ts [hunk 2]
```diff
@@ -169,49 +138,51 @@ export function truncatePathToDir(directory: string, pathName: string): string|
...
 
-    const absoluteBasePath =
-        truncatePathToDir(npmPackage.name, absoluteNpmMainPath);
+    // const absoluteBasePath =
```
Comment (e111077) [inline runner/config.ts]: please remove

**Review by e111077 — CHANGES_REQUESTED** [review]

**Comment by aomarks** [comment]
Do we still want this PR? cc @usergenic who is doing wct surgery right now.

**Comment by aomarks** [comment]
Closing this pull request because this repo is now archived. If this PR is still active, please submit a new one at https://github.com/Polymer/tools.

**Pull request closed without merging**